spec/integration/geo_config_spec.rb

```diff
@@ -64,6 +64,10 @@ def stub_mods_request(druid, body)
     it 'contains date' do
       expect(result['solr_year_i']).to eq [1603]
     end
+
+    it 'contains a reference to the source collection' do
```

Comment:
Should there be a test for non-geo types?

Reply:
Cool. I was confused by this being geo_config_spec.

Reply:
That's fair – geo_config is more accurately "earthworks_config" since it handles a bunch of things that aren't strictly `geo`.